spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/src/main/appengine/app.yaml

```diff
@@ -1,9 +1,5 @@
-runtime: java
-env: flex
+runtime: java11
+instance_class: F1
 
-handlers:
-- url: /.*
-  script: this field is required, but ignored
-
-manual_scaling:
-  instances: 1
+env_variables:
+  JAVA_TOOL_OPTIONS: "-XX:MaxRAM=256m -XX:ActiveProcessorCount=2 -Xmx32m"
```

Comment:
Is this based on experimentation or some guidance somewhere?

Reply:
Lifted unapologetically from https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/appengine-java11/springboot-helloworld/src/main/appengine/app.yaml#L21